contrib/endpoints/src/api_manager/context/request_context.cc

```diff
@@ -130,13 +130,14 @@ RequestContext::RequestContext(std::shared_ptr<ServiceContext> service_context,
 
 std::string RequestContext::GetRequestHTTPMethodWithOverride() {
   std::string method;
-  request_->FindHeader(kHttpMethodOverrideHeader, &method);
+  bool is_override_present = request_->FindHeader(kHttpMethodOverrideHeader,
```

Comment:
I don't understand what is "a statement with side effect".  But I feel the code is easier to read if it is:

// If not found method override
if (!FindHeader(METHOD-OVERRIDE, ) ) {
    Get normally.
}